src/siminterface/simulator.py

```diff
@@ -33,6 +34,12 @@ def __init__(self,  network_file, service_functions_file, config_file, resource_
         self.sf_list = reader.get_sf(service_functions_file, resource_functions_path)
         self.config = reader.get_config(config_file)
         self.metrics = Metrics(self.network, self.sf_list)
+        # Assume result path is the path where network file is in.
+        self.result_base_path = os.path.dirname(self.network_file)
+        if 'trace_path' in self.config:
+            # Quick solution to copy trace file to same path for network file as provided by calling algo.
+            trace_path = os.path.join(os.getcwd(), self.config['trace_path'])
```

Comment:
@stefanbschneider This is a quick fix that I applied here to get the file copied. It's not as generic as possible, and assumes the network file is placed in the result file, which is the case with our algorithms so far.